ipfs-cluster-service/main.go

```diff
@@ -261,7 +261,82 @@ removal, upgrade state using this command, and restart every peer.
 					Action: func(c *cli.Context) error {
 						err := upgrade()
 						checkErr("upgrading state", err)
-						return err
+						return nil
+					},
+				},
+				{
+					Name:  "export",
+					Usage: "save the IPFS Cluster state to a human readable json file",
+					Description: `
+
+This command reads the current cluster state and saves it as json for 
+human readability and editing.  In general any version of cluster will only
+work with the latest state format, so out of date snapshots will be migrated
+before the export.
+`,
+					Action: func(c *cli.Context) error {
+						err := export()
+						checkErr("exporting state", err)
+						return nil
+					},
+				},
+				{
+					Name:  "import",
+					Usage: "load an IPFS Cluster state from an exported human readable json file",
+					Description: `
+
+This command reads in a json encoding of the state specified in the argument
+storing the state as a persistent snapshot to be loaded as the cluster state
+when the cluster peer is restarted.  Paths are assumed to be relative to the
```

Comment:
hmm just open the given path directly. `ipfs-cluster-service import state.json` should read state.json from the local folder, regardless of the cluster base directory.